ingest/validation/metadata_validation.py

```diff
@@ -0,0 +1,161 @@
+"""Validate input metadata TSV file against metadata convention.
```

Comment:
Did you do this rename with `git mv`?  

File renames might be Git / GitHub's least well-supported common developer action.  Cases like this make file history in Git discontiguous, and much less feasible to review.

I ~~complained about~~ [explored Git renames](https://broadinstitute.slack.com/archives/C1C22V6FN/p1630673632002800) in #softeng a while ago.  I don't recommend we use the tool noted there; just context for the curious.  The [enhanced GitHub support](https://github.blog/changelog/2022-06-06-view-commit-history-across-file-renames-and-moves/) noted there doesn't seem to have applied here.  The effort to preserve history and enable easy code review in this case is probably not worthwhile.  

I'd just try `git mv` for future cases if you didn't here.  It's not certain to work, but seems like a better bet than raw `mv` or file renaming via editor.

Reply:
This wasn't a rename, I ran into an issue with circular imports. `validate_metadata.py` was supposed to be runnable on the command line. Ingest pipeline only calls functions from `validate_metadata` so the fact that `__main__ `had been changed to invoke `CellMetadata` (with no import from `cell_metadata.py`) meant it could no longer be run via command line to generate reference files to update tests.

To avoid the circular import problem, I duplicated `validate_metadata.py` and created `metadata_validation.py` which has only the command line and argument parsing capabilities that used to be in `validate_metadata.py`. This discussion made me realize I had neglected to update the metadata convention update README to use `metadata_validation.py` instead of `validate_metadata.py`... Going to update that now!